packages/configuration-builder/src/ColorsSection/ColorsSection.tsx

```diff
@@ -0,0 +1,34 @@
+import { useTranslation } from "react-i18next";
+import { ConfiguratorSection } from "../ConfiguratorSection/ConfiguratorSection";
+import { BrandColors } from "./BrandColors";
+import { ThemeConfig } from "../ThemeConfigurator/ThemeConfigurator";
+
+type ColorsConfig = ThemeConfig["colors"];
+
+type Props = {
+  value: ColorsConfig;
+  onChange: (value: ColorsConfig) => void;
+};
+
+export function ColorsSection(props: Props) {
+  const { t } = useTranslation();
+
+  return (
+    <ConfiguratorSection
+      title={t("ColorsSection.title")}
+      steps={[
+        { label: t("ColorsSection.Step.brand") },
+        { label: t("ColorsSection.Step.interactive") },
+        { label: t("ColorsSection.Step.neutral") },
+        { label: t("ColorsSection.Step.semantic") },
+        { label: t("ColorsSection.Step.dataVisualization") },
+      ]}
+      currentStep={0}
```

Comment:
Do we want this to be hardcoded for each step? Maybe we can move this up and make it automatic somehow?

Reply:
It's hardcoded in this PR since I implemented only the brand colors section. This component will handle all colors subsections in https://github.com/buildo/bento-design-system/pull/694/files#diff-0f0a67c12ad46eead5a0da483f0d53ab788ea850405e7fb8ef3e3398513b1bdcR52